src/chains.jl

```diff
@@ -364,21 +364,16 @@ Base.convert(::Type{Array}, chn::Chains) = convert(Array, chn.value)
 # timestamps.
 to_datetime(t::DateTime) = t
 to_datetime(t::Float64) = unix2datetime(t)
-to_datetime(t) = missing_datetime(typeof(t))
 to_datetime_vec(t::Union{Float64, DateTime}) = [to_datetime(t)]
 to_datetime_vec(t::DateTime) = [to_datetime(t)]
 to_datetime_vec(ts::Vector) = map(to_datetime, ts)
-to_datetime_vec(ts) = [missing]
+to_datetime_vec(ts) = missing
```

Comment:
This change seems a bit dangerous. AFAICT the method is supposed to return a vector, as in the cases above and currently done. It should also not be necessary for getting rid of the warnings. What was your motivation for this change?


```suggestion
to_datetime_vec(ts) = [missing]
```

Reply:
Those were the only calls `to_datetime_vec`, so the motivation was to avoid allocation `[missing]`.